packages/v4/patternfly-docs/content/accessibility/background-image/background-image.md

```diff
@@ -3,11 +3,36 @@ id: Background image
 section: components
 ---
 
-A **background image** allows you to place an image in the background of your page or area of a page.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-**Keyboard users** should not be able to interact with or navigate to the background image, as it is not an interactive element.
+## Accessibility
 
-**Screen reader users** do not need to be able to interact with the background image. However, using alternative text for the screen reader to be able to describe the background image when the user navigates to a page is beneficial.
+To implement an accessible PatternFly **background image**:
 
-## Accessibility application:
 - Check that there is sufficient color contrast between different parts of the background image and any text that will be on the image. Automated testing tools don't often pick up background image colors, so they may not be able to tell you if contrast is sufficient for users with visual disabilities.
```

Comment:
Could be worth adding this item as a testing checkbox as well so that it's in both sections. Doing that we could shorten this bullet by pulling out the second sentence ("Automated testing tools...") and placing it as the test item description.